deployment/roles/drone/vars/main.yml

```diff
@@ -1,32 +1,64 @@
 # Set this to values of an GitHub OAuth application created at
 # https://github.com/organizations/rchain/settings/applications
 #
-# drone_github_client: 
-# drone_github_secret: 
+drone_github_client: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          39303761343435326239663138396337616236346432336636616666323761353338383863346339
+          6236386265303666663436336437353863353334393764300a376336326637636462613337306530
+          34383261643864393161653765383335316532626234656431656636393638303132633932383938
+          3034373036383731360a393834643636366361653962666430623963396365303563663433383333
+          36303735383435623934633566613334393266386566653061303130303562313066
+drone_github_secret: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          39666665376238646264316437336331323636346432663534353536303762616531366166353838
+          3030363739656666613531323238616436316231343836650a323930393765653931353266346633
+          62653466633931356239663934393534363931313632386664333131346236646437663138393739
+          6665663064303231660a343431366337356238346239313538306535346230316164643932343333
+          36626633383330343165333431373437333733646536316237373332363232373037373239336363
+          3731633863653666383230373830646231376132386639323836
 
 # Set this to a random string. It's used to authenticate Drone agent with
 # server.
 #
-# drone_secret: 
+drone_secret: !vault |
+          $ANSIBLE_VAULT;1.1;AES256
+          36316365373366633336666338653937643964656261326630373939326264333932643636303264
+          6164343966626639303465636437303731306139643330380a653166386138643037373532656237
+          34336636356362336330633635653436356536373334366266623365323163373539343462653538
+          3433376462373636390a623461383866393231656462653235383931366562393934343462313034
+          36396635383766613436653133353665363866663930343561666430656662643630
 
 # Set this to comma separated list of GitHub organizations whose users can use
 # this Drone instance.
 #
-# drone_orgs: 
+drone_orgs: rchain,lukasz-golebiewski-org
```

Comment:
`lukasz-golebiewski-org` is it still needed?

Reply:
yes